src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printEcho(String command) {
+        System.out.println("____________________________________________________________");
+        System.out.println(command.substring(5));
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printToDo(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printDeadline(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printEvent(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printList(Task[] tasks, int size) {
+        System.out.println("____________________________________________________________");
+        for (int i = 0; i < size; i++) {
+            System.out.println(tasks[i].getStringNo() + "." + tasks[i].toString());
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void setTaskDone(Task task) {
+        task.markAsDone();
+        System.out.println("____________________________________________________________");
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("\t" + task.toString());
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printDone(Task[] tasks, int size, String command) {
+        String description = command.substring(5);
+        for (int i = 0; i < size; i++) {
+            if (description.equals(tasks[i].getDescription())){
+                setTaskDone(tasks[i]);
+                break;
+            }
+        }
+    }
+
+    public static void main(String[] args) {
+        printGreeting();
+        int size = 0;
+        Task[] tasks = new Task[100];
```

Comment:
You can extract 100 as MAX_NUMBER_OF_TASKS as other readers may not know how you derived the number 100

Reply:
It is better not to include the "magic number" in the code, eg. 100 here. It might be better to extract it as a variable and then use it in the codeing